components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-[2px]";
+
+export const Footer = ({ links }: FooterProps) => {
+  return (
+    <footer
+      className="flex items-center justify-center bg-blue pb-14 pt-10 md:pt-[52px]"
+      aria-label="Footer"
+      data-testid="footer"
+    >
+      <div className="mx-auto flex w-full max-w-[1040px] flex-col items-center justify-between md:flex-row">
+        <Link
```

Comment:
I cannot access with the keyboard to this element, and I don't see the point of having a Link with a logo to the home page, when later we have a link named Home.
IMO we should add `aria-hidden` to the logo container and make it accessible with the keyboard.

@Santiago-j-s could you help us here?

Reply:
> I cannot access with the keyboard to this element

@LPRegen I can, tried with Chrome and Firefox. Is it only this one? Weird.

---

>  I don't see the point of having a Link with a logo to the home page, when later we have a link named Home.

Remember that these links are just for "show", they may vary.

Reply:
> IMO we should add `aria-hidden` to the logo container and make it accessible with the keyboard.

`aria-hidden` breaks tests, hm.. maybe I should use a `nav` wrapper instead of a `div`?

Reply:
> @LPRegen I can, tried with Chrome and Firefox. Is it only this one? Weird.

Yes is the only one. In Firefox 'dev edition'(?) I wasn't able to select it but in Google Chrome I can.
I'll mark this as resolved then, considering that @TonyMckes can select it with the keyboard in Firefox and Chrome.